### PR TITLE
Add stat20 hub.

### PR DESCRIPTION
**Pull request by ryanlovett**

### CODE REVIEW

File: deployments/stat20/config/prod.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+nfsPVC:
+  nfs:
+    shareName: export/homedirs-other-2020-07-29/stat20/prod
```
Comment (yuvipanda) [inline deployments/stat20/config/prod.yaml:3]: you need to ssh into the NFS server and create this one.

**Comment by ryanlovett** [comment]
Not sure why stat159 is failing. It works locally, and the commits are unrelated to stat159.

File: deployments/stat20/image/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# RStudio support
+jupyter-rsession-proxy==1.4
```
Comment (yuvipanda) [inline deployments/stat20/image/requirements.txt]: We install a newer version explicitly in the dockerfile. Let's keep this or that?

**Review by yuvipanda — COMMENTED** [review]
this says the datahub/secrets/prod.yaml was deleted, that is probably unintentional?

Have you been able to deploy this manually?

File: deployments/stat20/config/prod.yaml (hunk 1 above)
Comment (ryanlovett) [reply deployments/stat20/config/prod.yaml:3]: Done, thanks. I figured I'd do this right before signing in the first time.

**Comment by ryanlovett** [comment]
datahub/secrets/prod.yaml wasn't deleted, but a lot of duplicate lines were removed. Those duplicates are due to a sops bugs.

I did a hubploy build manually, but I haven't tried to deploy yet. I'll try that.

File: deployments/stat20/image/requirements.txt (hunk 2 above)
Comment (ryanlovett) [reply deployments/stat20/image/requirements.txt]: Fixed, thanks!

Comment (ryanlovett) [reply deployments/stat20/image/requirements.txt]: @yuvipanda I had copied this over from the publichealth hub! Needs to be fixed there too.

**Comment by ryanlovett** [comment]
@yuvipanda I've deployed stat20-staging. I think I have to deploy datahub-staging as well so that it has the stat20 oauth vars.

**Comment by yuvipanda** [comment]
@ryanlovett yep, go for it.

**Comment by ryanlovett** [comment]
@yuvipanda stat20-staging is up!

**Pull request merged**